src/Symfony/Component/Messenger/Transport/Doctrine/Connection.php

```diff
@@ -35,6 +35,9 @@ class Connection
         'queue_name' => 'default',
         'redeliver_timeout' => 3600,
         'auto_setup' => true,
+        'pgsql_get_notify' => false,
```

Comment:
This should probably be its own transport, not mixing PostgreSQL-specifics with DBAL abstractions

Reply:
I had the same feeling in the beginning, but it will require to duplicate most of the code of this transport. So I'm not sure that it is worth it from a maintenance point of view.
Now that the implementation is less intrusive than it was, I'll check if the PostgreSQL related code can be moved to a decorator.

Reply:
Yeah, but that's not a big deal: more problematic is having generic SQL and PGSQL mixed within the same service, activated conditionally.

If the "enable norify" logic deals to a completely dedicated adapter, fixing specific bugs and maintenance becomes much easier, and so will debugging too.

Reply:
I isolated the PostgresSQL-specific code in a dedicated class, it's a bit cleaner, and while not perfect I think it will be good enough for now at least.